### ISSUE TITLE
UI request: make fold/unfold persistent

### ISSUE THREAD

**Issue by rpgoldman** [SynBioHub/synbiohub#1143]
One thing I'm finding that really hinders using the SynBioHub interface is that when I drill down into something -- i.e., click on a component of the network of a design -- I will be pushed to a new page.  That's fine.

What's bad is that when I back up, I get back to the original page for the design, only everything I had expanded is now collapsed.  And there are no breadcrumbs to tell me where I was before I went to the new page.

When I'm digging into a network, this makes things *really* hard to follow.  It would be great if SBH would just leave these things open.  Keep a setting on the page that knows what's open and closed.

If that's too hard (and it might be), how about giving me an option just render these pages without the peekaboo stuff: just show me the whole thing.  I'd rather scroll than have to play hide and seek...

Here's some related wisdom from Jakob Nielsen:
![image](https://user-images.githubusercontent.com/3274/73111863-965d3800-3ed1-11ea-8f63-1d34026e6a3a.png)

I'm definitely in the category of "users who parachute into very specific but inappropriate destinations through search or deep links." and could use some "rescue"!

**Comment by cjmyers** [comment]
I'm not sure the difficulty in leaving breadcrumbs.  @udp @3ach any idea how difficult this would be?  

It would certainly not be difficult to have menus default to be open, but there is a tradeoff here.  While it may not bother you to have a long page, the decision to close them was to avoid overwhelming users of SBH with a lot of information that they may not be interested in initially.  Would be good to get a sense of which type of user is more common.

**Comment by rpgoldman** [comment]
> It would certainly not be difficult to have menus default to be open, but there is a tradeoff here. While it may not bother you to have a long page, the decision to close them was to avoid overwhelming users of SBH with a lot of information that they may not be interested in initially. Would be good to get a sense of which type of user is more common.

Can we set a preference with a cookie? That seems like the obvious work-around if there are users with different preferences.

I don't think we can associate with an account because, for example, the entire SD2 program seems to have one account!

**Comment by cjmyers** [comment]
Not sure.  Probably.  Outside my wheelhouse though.  @udp @3ach thoughts?
